src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -178,3 +178,8 @@ def test_vi_int32_output_function(session):
     assert isinstance(backlog, int)
     assert backlog == 0
 
+
+def test_initialize_nondefault():
+    with nidmm.Session('FakeDevice', False, False, 'Simulate = 1,RangeCheck=0,QueryInstrStatus=1,Cache=0,DriverSetup=Model:4080; BoardType:PXIe') as session:
+        assert session.simulate is True

```

Comment:
Shouldnt we also assert on the other items (false, false, device name, etc)

Reply:
I don't think we can assert whether Reset_device is true or false
And for Simulated devices, ID Query will not be returned, an exception will be thrown instead (can add a system test for this if needed?), so we cant assert that either.
Another system test already checks device name assertion so it can be omitted.